README.adoc

```diff
@@ -259,11 +262,20 @@ information that is received from the Server-Sent Events.
 === Subscribing to SSE
 
 The [hotspot=initSSE file=0]`initSSE()` method is called when the page first
-loads. This subscribes the client to the SSE by creating a new instance of
-[hotspot=eventSource file=0]`EventSource` interface and specifying the `/bff/sse`
-endpoint in the parameter. The [hotspot=eventSource file=0]`EventSource` interface
-makes a `GET` request to this endpoint with a request header of `Accept:
-text/event-stream` to connect to the server.
+loads. This subscribes the client to the SSE by creating a new instance of the
+[hotspot=eventSource file=0]`EventSource` interface and specifying the
+`http://localhost:9084/bff/sse` URL in the parameters. The [hotspot=eventSource
+file=0]`EventSource` interface makes a `GET` request to this endpoint with a
+request header of `Accept: text/event-stream` to connect to the server. 
+
+Because this request is coming from `localhost:9080` and being made to
```

Comment:
**change**
Because this request is coming from `localhost:9080` and being made to
`localhost:9084` it's considered to be cross-origin and must follow the Cross-Origin
Resource Sharing (CORS) specification in order to not be blocked by the browser.

**to**
Because this request comes from `localhost:9080` and is made  to
`localhost:9084`, it  must follow the Cross-Origin
Resource Sharing (CORS) specification to avoid being blocked by the browser.